core/main/src/processor/storage/storage_manager.rs

```diff
@@ -76,11 +76,32 @@ pub struct StorageManager;
 
 impl StorageManager {
     pub async fn get_bool(state: &PlatformState, property: StorageProperty) -> RpcResult<bool> {
+        if property.is_a_privacy_setting_property() {
+            // check if the privacy setting property is available in cache
+            let privacy_settings_cache = state.ripple_cache.get_privacy_settings_cache();
+            let val_opt = property.get_privacy_setting_value(&privacy_settings_cache);
+            if let Some(val) = val_opt {
+                return Ok(val);
+            }
+        }
+
```

Comment:
This should probably be made generic so privacy property-specific conditionals don't have to be inserted throughout `StorageManager`, the added bonus is that any arbitrary property would have cache support if we want. E.g. instead of `RippleCache::get_privacy_settings_cache()` you could have `RippleCache::get_cache(property)` and let `RippleCache` figure out if the particular property is cached.

Reply:
@pahearn73 : Thanks for the review comment. 
RippleCache now handles the identification of cache-able properties and performs set/get values to/from the cache.